extensions/sql-migration/src/dashboard/dashboardTab.ts

```diff
@@ -347,27 +347,27 @@ export class DashboardTab extends TabBase<DashboardTab> {
 			{
 				title: loc.DASHBOARD_HELP_LINK_MIGRATE_USING_ADS,
 				description: loc.DASHBOARD_HELP_DESCRIPTION_MIGRATE_USING_ADS,
-				link: 'https://docs.microsoft.com/azure/dms/migration-using-azure-data-studio'
+				link: 'https://learn.microsoft.com/azure/dms/migration-using-azure-data-studio'
```

Comment:
Can we replace all of these with aka.ms links across the board -- perhaps this PR is not the right place to do it, but maybe we can spin up an ADO task to do this in the next release... any hardcoded links (especially Microsoft docs) seem quite easy to break...

Reply:
I would recommend we do this in another task for all links